system_baseline/models.py

```diff
@@ -106,23 +110,22 @@ class SystemBaselineMappedSystem(db.Model):
 
     id = db.Column(UUID(as_uuid=True), primary_key=True, default=uuid.uuid4)
     account = db.Column(db.String(10), nullable=False)
+    org_id = db.Column(db.String(36))
     system_baseline_id = db.Column(
         UUID(as_uuid=True), ForeignKey("system_baselines.id"), nullable=False
     )
     system_id = db.Column(UUID(as_uuid=True), nullable=False, index=True)
 
     @classmethod
-    def delete_by_system_ids(cls, system_ids, account_number):
-        cls.query.filter(cls.system_id.in_(system_ids), cls.account == account_number).delete(
-            synchronize_session="fetch"
-        )
+    def delete_by_system_ids(cls, system_ids, account_number, org_id):
+        query = cls.query.filter(cls.system_id.in_(system_ids), cls.account == account_number)
+        query.delete(synchronize_session="fetch")
         db.session.commit()
 
     @classmethod
-    def get_mapped_system_count(cls, account_number):
-        return (
-            cls.query.with_entities(cls.system_baseline_id, func.count(cls.system_baseline_id))
-            .group_by(cls.system_baseline_id)
-            .filter(cls.account == account_number)
-            .all()
-        )
+    def get_mapped_system_count(cls, account_number, org_id):
```

Comment:
same question as on line 120 above re: using org_id in the query

Reply:
You are right, I missed this. Thank you.